fccloudapi.php

```diff
@@ -171,6 +171,449 @@ public function getObjectName()
     }
 }
 
+/**
+ * Base MetadataRecord class
+ * @package codelathe\fccloudapi
+ */
+abstract class AbstractMetadataRecord extends DataRecord
```

Comment:
Why is that named the AbstractMetadataRecord? This is in fact the attribute value converter.

Reply:
You may be right. I investigated making the change but it seems the current way makes testing a bit easier so I'd like to keep it like this for now.